### ISSUE TITLE
Fix invalid cached states which are making `lightwalletd` tests fail

### ISSUE THREAD

**Issue by mpguerra** [ZcashFoundation/zebra#7661]
## Motivation

It looks like the lightwalletd tests are failing because a previous lightwalletd job didn't write a full cache:

```
{"app":"lightwalletd","level":"info","msg":"Done reading 88745 blocks from disk cache","time":"2023-09-30T21:11:05Z"}
{"app":"lightwalletd","level":"info","msg":"Adding block to cache 509062
```

https://github.com/ZcashFoundation/zebra/actions/runs/6365517114/job/17282788077?pr=7636#step:9:228

## Suggested Fix

- [x] Find all the lightwalletd cached states with heights less than 2 million
- [ ] Work out which PR, branch, and CI job created them, and open a ticket to fix it
- [ ] Optional: stop creating new invalid images, by stopping running CI on that PR, or disabling that scheduled CI job (if that's possible)
- [x] Delete all the lightwalletd cached states with heights less than 2 million (they are all invalid)
- [ ] Repeat these steps until the original bug is fixed, merged, and all PRs are updated with that fix

### Background

This ticket has some notes about how we fixed this last time: [dev docs: Document how to fix cached state issues](https://github.com/ZcashFoundation/zebra/issues/7558)

**Comment by gustavovalverde** [comment]
Deleted this disk created by:
- https://github.com/ZcashFoundation/zebra/commit/6f50304
```
Description: Created from commit 6f50304 with height 506458 and database format v25-1-1
Selected Disk: lwd-cache-v1-2-0-6f50304-v25-mainnet-tip-031355
```

**Comment by teor2345** [comment]
Since this cached state was created by Zebra `v1.2.0`, and we've already fixed bug #7335 in the `main` branch, we don't need to do anything more here.
